collections/esdc-8d-0.083deg-1x2160x4320-2.1.1.yaml

```diff
@@ -0,0 +1,202 @@
+Name: esdc-8d-0.083deg-1x2160x4320-2.1.1
```

Comment:
This doesn't seem user-friendly. Could you provide a more specific name than the `DataID`, or is this collection name a requirement?